pyproject.toml

```diff
@@ -20,15 +20,21 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.8 <3.13"
-fastapi = ">=0.70.0"
+fastapi = { version = ">=0.70.0", optional = true }
+fastapi-slim = { version = ">=0.111.0", optional = true }
 injector = ">=0.19.0"
 
 [tool.poetry.group.dev.dependencies]
+fastapi-slim = ">=0.111.0"
```

Comment:
I used `fastapi-slim` as it's the lighter version. Also, considering before was marked as `>=0.70.0`, we'd be taking the latest version anyway.